app/src/main/java/uk/gov/android/securestore/crypto/limitedmanager/LimitedCryptoManager.kt

```diff
@@ -0,0 +1,49 @@
+package uk.gov.android.securestore.crypto.limitedmanager
+
+/**
+ * Interface to handle encryption and decryption of [String] data
+ */
+interface LimitedCryptoManager {
```

Comment:
Unsure of the name of this class, mainly the "limited" part.

Reply:
Changed it now since I had refactored it to be separate from the Hybrid one so is now called `SymmetricCryptoManager`